app/src/main/kotlin/com/wire/android/ui/calling/ongoing/fullscreen/FullScreenTile.kt

```diff
@@ -87,7 +89,7 @@ fun FullScreenTile(
                     .height(height)
                     .padding(contentPadding),
                 participantTitleState = it,
-                isSelfUser = selectedParticipant.isSelfUser,
+                isSelfClient = selectedParticipant.clientId == participants[0].clientId,
```

Comment:
why did you change it? Os there any reason to not use `SelectedParticipant.isSelfUser`?
1. in case of design changes and self user is not always in a first tile we'll have to make changes here too.
2. It's preferred to as less logic in UI as possible.

Reply:
When you join the same call from two devices then you get two participants with `isSelfUser` flag set to true. So these users have incorrectly duplicated mute and flip camera buttons and video is only showing for one of them.
Needed to use a clientId to handle them correctly.

Reply:
so maybe lets update `SelectedParticipant` by changing `isSelfUser` to `isSelfClient` or just adding `isSelfClient` and using it from there?

Reply:
I am trying to fetch the clientId in the viewmodel. I think to make it correct we can add 'isSelfUser' flag to the UICallParticipant and use clientId to set it when mapping to UIParticipant. This way the flag will be correct across all UI code and will not depend on the participants ordering logic.

Reply:
@borichellow please check updated logic for isSelfUser based on clientId.